.github/workflows/ci.yml

```
@@ -1,95 +1,61 @@
 name: CONTINUOUS INTEGRATIONS
-
 on:
-  pull_request:
-  workflow_dispatch:
-
+  push:
+    branches-ignore:
+      - '!main'
```

Para rodar só na main basta fazer isso:
```suggestion
  push:
    branches: main
```

---

alterei no: https://github.com/uLucasFraga/cypress_api_for_studies/pull/27/commits/4d5589b7f6b5cb4f055d0c96860e98c94c79387a